### ISSUE TITLE
Infer Base URL Based on Current Date

### ISSUE THREAD

**Issue by rye** [StoDevX/cs251-toolkit#16]
In `cs251tk/toolkit/args.py`, would it make sense to infer the default base URL based on the current date?

**Comment by hawkrives** [comment]
Absolutely. 

StoGit uses a $course-$semester$calendarYear format: "sd-s14" (software, spring of 2013-14), "hd-f17" (hardware, fall of 2017-18). 

So really, we'd assume software design (for now), say that semester is "s" if currentMonth < 6, otherwise "f", and just grab the calendar year?

I think that should work.

**Comment by rye** [comment]
Could we include it in the spec somehow?

**Comment by hawkrives** [comment]
Then we'd have to update each spec file every year, though. 

I could see benefits for making a "class spec" file - containing, like, the StoGit base, and a list of homeworks, maybe including due dates. 

But I don't think I want to include the stogit base URL in the current spec files, no.

**Comment by hawkrives** [comment]
```python
now = datetime.date.today()
abbr = 'sd'
semester = 's' if now.month < 6 else 'f'
year = str(now.year)[2:]
stogit = 'git@stogit.cs.stolaf.edu'
base_url = '{}:{}-{}{}'.format(stogit, abbr, semester, year)
```

I think that should work? Needs testing.

**Comment by hawkrives** [comment]
I have a commit that should do this locally. I'll PR it soon.

**Comment by hawkrives** [comment]
Closed by #36.
